nibiru/assetlist.json

```diff
@@ -43,6 +43,42 @@
       ],
       "type_asset": "sdk.coin"
     },
+    {
+      "description": "Liquid Staked Nibiru (Eris)",
+      "extended_description": "Liquid Staked Nibiru, powered by Eris Protocol's amplifier contracts. Nibiru is a smart contract ecosystem with a high-performance, EVM-equivalent execution layer. Nibiru is engineered to meet the growing demand for versatile, scalable, and easy-to-use Web3 applications.",
+      "socials": {
+        "website": "https://nibiru.fi/docs/learn/liquid-stake/",
+        "twitter": "https://x.com/eris_protocol"
+      },
+      "denom_units": [
+        {
+          "denom": "tf/nibi1udqqx30cw8nwjxtl4l28ym9hhrp933zlq8dqxfjzcdhvl8y24zcqpzmh8m/ampNIBI",
+          "exponent": 0
+        },
+        {
+          "denom": "stNIBI",
+          "exponent": 6
+        }
+      ],
+      "base": "tf/nibi1udqqx30cw8nwjxtl4l28ym9hhrp933zlq8dqxfjzcdhvl8y24zcqpzmh8m/ampNIBI",
+      "name": "Liquid Staked Nibiru (Eris)",
+      "display": "stNIBI",
+      "symbol": "stNIBI",
```

Comment:
Please define the trace for this, showing how it relates to NIBI (using trace type: liquid-stake)

Reply:
Updated in 5f8a4e8ee8cfceca0a8d914b5623e3876f0175ff